contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3713,7 +3713,7 @@ CREATE OR REPLACE FUNCTION sys.is_rolemember_internal(
 	IN database_principal sys.SYSNAME
 )
 RETURNS INT AS 'babelfishpg_tsql', 'is_rolemember'
-LANGUAGE C STABLE PARALLEL SAFE;
+LANGUAGE C STABLE PARALLEL UNSAFE;
```

Comment:
We need to add these in upgrade files as well, right?

Reply:
Updated in latest rev.